aspnetcore/signalr/hubs.md

```diff
@@ -38,6 +38,7 @@ Create a hub by declaring a class that inherits from <xref:Microsoft.AspNetCore.
 > Hubs are [transient](/dotnet/core/extensions/dependency-injection#transient):
 >
 > * Don't store state in a property of the hub class. Each hub method call is executed on a new hub instance.
+> * Don't instantiate a hub directly via dependency injection. Hubs aren't intended to be used directly. Only interact with a hub via an [`IHubContext`](xref:signalr/hubcontext).
```

Comment:
> Only interact with a hub via an [`IHubContext`]

You aren't interacting with a hub, you're getting an interface that allows you to interact with client(s).

Reply:
Can I merely strike that sentence?

Reply:
@BrennanConroy is there value to what he's trying to write, ie **Don't instantiate a hub directly via dependency injection**

If so, can you use the suggestion menu to suggest better wording? I'll apply that wording throughout and we can merge this PR.